components/dashboard/src/components/UsageView.tsx

```diff
@@ -159,261 +135,309 @@ function UsageView({ attributionId }: UsageViewProps) {
-                            onChange={(date) => date && setEndDate(dayjs(date))}
-                            selectsEnd
-                            startDate={startDate.toDate()}
-                            endDate={endDate.toDate()}
-                            minDate={startDate.toDate()}
-                            customInput={<DateDisplay />}
-                            dateFormat={"MMM d, yyyy"}
-                        />
-                    </div>
-                }
-            />
+            <Header title="Usage" subtitle="updated every 15 minutes" />
             <div className="app-container pt-5">
                 {errorMessage && <p className="text-base">{errorMessage}</p>}
-                {!errorMessage && (
-                    <div className="flex space-x-16">
-                        <div className="flex">
-                            <div className="space-y-8 mb-6" style={{ width: "max-content" }}>
-                                <div className="flex flex-col truncate">
-                                    <div className="text-base text-gray-500 truncate">Current Month</div>
-                                    <div
-                                        className="text-sm text-blue-500 hover:text-blue-600 dark:text-blue-400 dark:hover:text-blue-500 truncate cursor-pointer mb-5"
-                                        onClick={() => handleMonthClick(startOfCurrentMonth, dayjs())}
-                                    >
-                                        {dayjs(startOfCurrentMonth).format("MMMM YYYY")}
-                                    </div>
-                                    <div className="text-base text-gray-500 truncate">Previous Months</div>
-                                    {getBillingHistory()}
-                                </div>
-                                {!usagePage.isLoading && (
-                                    <div>
-                                        <div className="flex flex-col truncate">
-                                            <div className="text-base text-gray-500">Credits</div>
-                                            <div className="flex text-lg text-gray-600 font-semibold">
-                                                <span className="dark:text-gray-400">
-                                                    {usagePage.data?.creditsUsed.toLocaleString()}
-                                                </span>
-                                            </div>
-                                            {currentOrg.data && currentOrg.data.isOwner && (
-                                                <div className="flex text-xs text-gray-600">
-                                                    <span className="dark:text-gray-500 text-gray-400">
-                                                        <Link to="/billing" className="gp-link">
-                                                            View Billing →
```

Comment:
**suggestion**: We can keep this around until we figure out a better way to incorporate with new designs for this page. A boring (simple) solution to keep it with the layout suggestion made earlier could be to use a second secondary button on the top right, near the upcoming Export as CSV button.

| View Billing link in usage |
|-|
| <img width="1440" alt="view-link" src="https://github.com/gitpod-io/gitpod/assets/120486/d1ac58de-0b3d-45b4-a4c6-cb7d2b76c26f"> |